WalletWasabi.Fluent/ViewModels/Dialogs/ShuttingDownViewModel.cs

```diff
@@ -1,31 +1,43 @@
+using ReactiveUI;
 using System.Reactive;
 using System.Reactive.Disposables;
 using System.Reactive.Linq;
 using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
 
 namespace WalletWasabi.Fluent.ViewModels.Dialogs;
 
-[NavigationMetaData(Title = "Wasabi is shutting down...")]
+[NavigationMetaData(Title = "Please wait to shut down...")]
 public partial class ShuttingDownViewModel : DialogViewModelBase<Unit>
 {
 	private readonly ApplicationViewModel _applicationViewModel;
+	private IDisposable? _intervalDisposable;
 
 	public ShuttingDownViewModel(ApplicationViewModel applicationViewModel)
 	{
 		_applicationViewModel = applicationViewModel;
+
+		NextCommand = ReactiveCommand.Create(OnNext);
 	}
 
 	protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disposables)
 	{
-		Observable.Interval(TimeSpan.FromSeconds(3))
-				  .Subscribe(_ =>
-				  {
-					  if (_applicationViewModel.CanShutdown())
+		_intervalDisposable =
+			Observable.Interval(TimeSpan.FromSeconds(3))
+					  .ObserveOn(RxApp.MainThreadScheduler)
+					  .Subscribe(_ =>
 					  {
-						  Close();
-						  _applicationViewModel.ShutDown();
-					  }
-				  })
-				  .DisposeWith(disposables);
+						  if (_applicationViewModel.CanShutdown())
+						  {
+							  Close();
+							  _applicationViewModel.ShutDown();
+						  }
+					  })
+					  .DisposeWith(disposables);
+	}
+
+	private void OnNext()
+	{
+		_intervalDisposable?.Dispose();
```

Comment:
I don't think this is needed.
When you call `Close()` the dialog will close and that also means an "away navigation" so dispose will be called on `disposables`.

Reply:
@soosr fixed in latest commit.